woocommerce/woocommerce
# Moving section icon svgs to assets folder and loading by URL

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Instead of injecting SVG directly into template, we're copying them as files into the assets folder and loading by URL.

Closes #37711 .



### How to test the changes in this Pull Request:

1. Run `pnpm --filter=woocommerce/client/admin run build`
2. Check that the `woocommerce/assets/client/admin` directory contains a `product-editor/icons` directory with all the previously built assets.
3. Run site, and enable `product-block-editor` feature flag via WCA Test Helper plugin.
4. Navigate to Products -> Add New to view product block editor.
5. Ensure the icons appear for the section headers in each section (Basic, Images, Pricing, Inventory, Shipping)
